src/universal/.devcontainer/devcontainer.json

```
@@ -39,8 +39,8 @@
             "version": "latest"
         },
         "ghcr.io/devcontainers/features/ruby:1": {
-            "version": "3.2",
-            "additionalVersions": "3.1"
+            "version": "3.3.3",
```

```suggestion
            "version": "3.3.4",
```

Can we update to 3.3.4?  Hoping to not run into a similar issue as https://github.com/devcontainers/images/issues/1123